### ISSUE TITLE
Change the license?

### ISSUE THREAD

**Issue by kevinyamauchi** [alisterburt/napari-workshops#10]
Hey @alisterburt . I am wondering if it makes sense to change the license for these materials to one less "software-y". Perhaps something like [CC-BY](https://creativecommons.org/licenses/by/4.0/)?

**Comment by kevinyamauchi** [comment]
Also, I think we should try to put the license somewhere on the pages so (or sidebar?) so people know they can re-use the materials.

**Comment by alisterburt** [comment]
Yep that sounds totally reasonable/smart - I don't know anything about the different licenses so very happy to defer to your judgement here. A link from the TOC so that it shows in the sidebar sounds like a welcome addition!
